ksp_plugin/planetarium.cpp

```diff
@@ -184,10 +180,14 @@ RP2Lines<Length, Camera> Planetarium::PlotMethod2(
       }
       Position<Navigation> const extrapolated_position =
           previous_position + previous_velocity * Δt;
-
-      position = plotting_frame_->ToThisFrameAtTime(t).rigid_transformation()(
+      to_plotting_frame_at_t = plotting_frame_->ToThisFrameAtTime(t);
+      position = to_plotting_frame_at_t.rigid_transformation()(
                      trajectory.EvaluatePosition(t));
 
+      // The quadratic term of the error between the linear interpolation and
+      // the actual function is maximized halfway through the segment, so it is
```

Comment:
And because it is an interpolation (the extrapolation also approximates to 2nd order, but has nonvanishing error at the end).